### PR TITLE
add(main): new package gomp

### PR DESCRIPTION
**Pull request by T-Dynamos**
Screenshot_20221231-103719_Termux

### CODE REVIEW

**Review by 2096779623 — CHANGES_REQUESTED** [review]
image

Hardcoded path here: https://github.com/aditya-K2/gomp/blob/0c0cbbf950c84f44092dbeb70d3ae2c095b79b9e/config/config.go#L116-L117

**Review by 2096779623 — CHANGES_REQUESTED** [review]
It should depend on `mpd` .

**Comment by 2096779623** [comment]
Screenshot_20221231-204434

It finally worked after I searched for the play button for half an hour....

**Review by PraveenNanda124 — COMMENTED** [review]
Great work

File: packages/gomp/build.sh [hunk 1]
```diff
@@ -0,0 +1,21 @@
+TERMUX_PKG_HOMEPAGE=https://aditya-k2.github.io/gomp/
+TERMUX_PKG_DESCRIPTION="MPD client inspired by ncmpcpp with builtin cover-art view and LastFM integration"
+TERMUX_PKG_LICENSE="GPL-3.0"
+TERMUX_PKG_MAINTAINER="@termux"
+_COMMIT=0c0cbbf950c84f44092dbeb70d3ae2c095b79b9e
+_COMMIT_DATE=2022.12.31
+TERMUX_PKG_VERSION=${_COMMIT_DATE//./}
+TERMUX_PKG_SRCURL=git+https://github.com/aditya-K2/gomp
```
[the remark this answers is not in the document]
Comment (T-Dynamos) [reply packages/gomp/build.sh:8]: > The Git URL specification isn't valid either. Must be a `https://github.com/aditya-K2/gomp.git`
Ah this changed, see PR #14281

**Pull request merged**